person.rb

```diff
@@ -0,0 +1,24 @@
+# Class that represents a person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] You could even use a string for `parent_permission` instead of a symbol
```suggestion
  def initialize(age, specialization, name = 'Unknown', parent_permission = true)
```